charts/prometheus-pushgateway/Chart.yaml

```
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "v1.11.0"
+appVersion: "v1.11.1"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
 version: 3.0.0
```

This change is to be considered minor.
```suggestion
version: 3.1.0
```